{"uuid": "7cbcb3ee-fbc2-4a45-aeec-f0933276e016", "children": ["09499c10-26e1-4fbe-b4e0-f19f0ee6f2d6"], "befores": [{"name": "_xunit_setup_class_fixture_TestWeCom", "status": "broken", "statusDetails": {"message": "IndexError: tuple index out of range\n", "trace": "  File \"D:\\python3\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"D:\\python3\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"D:\\python3\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\python3\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\python3\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"D:\\python3\\Lib\\site-packages\\_pytest\\python.py\", line 787, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, cls)\n  File \"D:\\python3\\Lib\\site-packages\\_pytest\\python.py\", line 715, in _call_with_optional_argument\n    func(arg)\n  File \"D:\\开发项目\\selenium_test_WeCom\\test_case\\test_wecom.py\", line 30, in setup_class\n    self.home = LoginPage().reuse_cookies_login()\n                ^^^^^^^^^^^\n  File \"D:\\开发项目\\selenium_test_WeCom\\page_obj\\auto_exception_screen_shot.py\", line 26, in inner\n    driver = args[0].driver\n             ~~~~^^^\n"}, "start": 1729254952247, "stop": 1729254957586}], "start": 1729254952247, "stop": 1729254957667}